contracts/PricerRule.sol

```diff
@@ -0,0 +1,408 @@
+
+    /**
+     * @dev Conversion rate from the base currency to the economy token.
+     */
+    uint256 public conversionRateFromBaseCurrencyToToken;
+
+    /**
+     * @dev Conversion rate decimals from the base currency to the
+     *      economy token.
+     */
+    uint256 public conversionRateDecimalsFromBaseCurrencyToToken;
+
+    /**
+     * @dev Token rules address of the economy.
+     */
+    TokenRules private tokenRules;
+
+    /**
+     * @dev Mapping from pay-currency code to corresponding price oracle.
+     *      The mapped price oracle's base currency code should be the economy
+     *      base currency code (see baseCurrencyCode storage variable)
+     *      and the quote currency code should be the corresponding pay-currency
+     *      code.
+     */
+    mapping(bytes3 => PriceOracleInterface) private baseCurrencyPriceOracles;
+
+    /**
+     * @dev Mapping from pay-currency code to price difference accepatance
+     *      margin. During a pay operation an intended price for
+     *      pay-currency is presented, that is checked against current price of
+     *      pay-currenency wrt stored acceptance margin.
+     */
+    mapping(bytes3 => uint256) private baseCurrencyPriceAcceptanceMargins;
+
+
+    /* Special Functions */
+
+    /**
+     * @notice Constructs a new pricer object.
+     *
+     * @dev Function requires:
+     *          - The economy token address is not null.
+     *          - The base currency code is not empty.
+     *          - Conversion rate from the base currency to the token is not 0.
+     *          - Conversion rate's decimals from the base currency to the
+     *            token is not 0.
+     *          - The economy token rules address is not null.
+     *
+     * @param _organization Organization address.
+     * @param _eip20Token The economy token address.
+     * @param _baseCurrencyCode The economy base currency code.
+     * @param _conversionRate The conversion rate from the economy base currency
+     *                        to the token.
+     * @param _conversionRateDecimals The conversion rate's decimals from the
+     *                                economy base currency to the token.
+     * @param _tokenRules The economy token rules address.
+     */
+    constructor(
+        OrganizationInterface _organization,
+        address _eip20Token,
+        bytes3 _baseCurrencyCode,
+        uint256 _conversionRate,
+        uint8 _conversionRateDecimals,
+        address _tokenRules
+    )
+        Organized(_organization)
+        public
+    {
+        require(_eip20Token != address(0), "EIP20 Token address is null.");
+
+        require(
+            _baseCurrencyCode != bytes3(0),
+            "Base currency code is null."
+        );
+
+        require(
+            _conversionRate != 0,
+            "Conversion rate from the base currency to the token is 0."
+        );
+
+        require(
+            _conversionRateDecimals != 0,
```

Comment:
0 should be an acceptable value for `conversionRateDecimals`, otherwise we are disallowing whole number conversion rates (e.g., 1:1, 1:2, etc.).

Reply:
Yes. I've fixed that already locally. Thx for catching!